Automattic/wp-calypso
# Remove Plan overlap notice

Related to pbNhbs-6rL-p2#comment-15072

## Proposed Changes

* Remove the plan overlap notice now that we'll be able to upgrade from products to plans

Before | After
---|---
<img width="920" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/cb6fa16d-1fa3-405a-ac28-d6c846b6ecb7"> | <img width="916" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/fb63a0c3-9eaa-485d-9dba-11ee28ff86f3">


## Testing Instructions



* Apply D110042-code to your sandbox if that hasn't been deployed yet
* You can follow the test plan in the patch above. The following is a summarized test plan:
* Spin up a JN site and purchase a Backup T1 plan
* Go to the Calypso live link, visit `/checkout/{JN_site_slug}/jetpack_security_t1_yearly`
* Confirm that you don't see the plan overlap notice, and that the price of the Backup plan is applied as a discount


## Pre-merge Checklist
